### DISCUSSION TITLE
Cefsharp cannot display when use UserControl

### DISCUSSION THREAD

**Discussion by congthien77** [cefsharp/CefSharp#3756] [General]
I use C# Wpf to build my application 
I add CefSharp to the UserControl, load the html file to display the content. However, the content of the html file is not displayed, even though the data is fully loaded
When loading a web page into UserControl through CefSharp, the web page is displayed 
When I replace UserControl with Window, the html file content is displayed 
Is there any special setting for CefSharp to display html files on UserControl?

**Comment by amaitland** [comment]
The main WPF example uses a UserControl, there are no special requirements. 

https://github.com/cefsharp/CefSharp/blob/cefsharp/92/CefSharp.Wpf.Example/Views/BrowserTabView.xaml
https://github.com/cefsharp/CefSharp/blob/cefsharp/92/CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs

You'll need to provide an example that reproduces the problem

**Reply by congthien77** [reply]
Thanks for the help.
I have fixed this problem. However, I am having the following problem:
I want to put the processing from Views to ViewModels in accordance with MVVM architecture standards, but CefSharp needs to call the event IsBrowserInitializedChanged before loading the web address.
I move the IsBrowserInitializedChanged event to ViewModels in the following way:
WebBrowser.IsBrowserInitializedChanged += Initialized;
private Initialized(sender sender, EventArgs e)
{
}
But the Initialized function is not called
Is it possible to use the UserControl to add the event IsBrowserInitializedChanged by the above method?

**Reply by amaitland** [reply]
You can bind to the Address property

**Reply by congthien77** [reply]
Thank you, I fixed it
One more problem, is there a way to bind the JavascriptObjectRepository.Register function from Views to ViewModel?

**Reply by amaitland** [reply]
You should be able to use an event to command 

http://cefsharp.github.io/api/91.1.x/html/E_CefSharp_IJavascriptObjectRepository_ResolveObject.htm

**Reply by congthien77** [reply]
Can you show me code example how to use event to command ?
